lib/tasks/safe_reset.rake

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+namespace :db do
+  desc "Drop, create, migrate then seed the development database"
+  task safe_reset: :environment do
+    if Rails.env.development? || Rails.env.deployed_development?
+      connection = ActiveRecord::Base.connection
+      tables = connection
+                   .execute("SELECT * FROM pg_catalog.pg_tables;")
+                   .filter { |row| row["schemaname"].include?("public") }
+                   .map { |row| row["tablename"] }
+      tables.delete "schema_migrations"
```

Comment:
why do we need to delete this?

Reply:
Looks like this removes that table from the list, since we don't want to run all the migrations again since the tables themselves aren't changing, just being truncated. Therefore we don't want to truncate `schema_migrations`. This looks correct to me

Reply:
Ethan is right. We want to remove all contents of our tables, but `migrations` are a bit 'meta'. Not really our table.